include/eosio/ship_protocol.hpp

```diff
@@ -266,7 +266,46 @@ namespace eosio { namespace ship_protocol {
    EOSIO_REFLECT(action_trace_v1, action_ordinal, creator_action_ordinal, receipt, receiver, act, context_free, elapsed,
                  console, account_ram_deltas, except, error_code, return_value)
 
-   using action_trace = std::variant<action_trace_v0, action_trace_v1>;
+   struct sync_call_trace {
```

Comment:
It feels like we ought to version this like everything else in ship. (sync_call_trace_v0)